oci/nginx/image.yaml

```diff
@@ -0,0 +1,6 @@
+version: 1
+
+upload:  
+  - source: https://git.launchpad.net/~ubuntu-docker-images/ubuntu-docker-images/+git/nginx
```

Comment:
can't we use more ssh url?

Reply:
more? like more than 1? Why?

Reply:
oh no like instead of https use git+ssh

Reply:
but then you'd need to add credentials to the OCI factory

Reply:
yeah true